Platform/ElkhartlakeBoardPkg/Script/BtgSign.py

```diff
@@ -168,11 +168,11 @@ def gen_oem_key_hash(output_dir, oem_pub_key, oem_pub_key_hash_file, hash_type,
     key_sz_bytes = int(key_size)//8
 
     if os.name == 'nt':
-        openssl_path = get_openssl_path() + '.exe'
+        openssl_path = get_openssl_path()
     else:
         openssl_path = 'openssl'

```

Comment:
Check if line 170 ~ 173 needed, similar to above similar place.
I assume get_openssl_path() will return correct value and line 175 you don't use openssl_path  anymore.

Reply:
Done updating it